src/main/kotlin/ecommerce/infrastructure/JwtTokenProvider.kt

```diff
@@ -0,0 +1,65 @@
+package ecommerce.infrastructure
+
+import ecommerce.entities.Member
+import io.jsonwebtoken.JwtException
+import io.jsonwebtoken.Jwts
+import io.jsonwebtoken.security.Keys
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+import java.nio.charset.StandardCharsets
+import java.util.Date
+import javax.crypto.SecretKey
+
+@Component
+class JwtTokenProvider(
+    @Value("\${security.jwt.token.secret-key}")
+    secret: String,
+    @Value("\${security.jwt.token.expire-length}")
+    private val validityInMs: Long,
+) {
+    private val secretKey: SecretKey =
+        Keys.hmacShaKeyFor(secret.toByteArray(StandardCharsets.UTF_8))
+
+    fun createToken(
```

Comment:
(Just ask! feel free 😄 )
What's different between infrastructure package and config package ?